docs/roles/frontend.md

```diff
@@ -1,48 +1,52 @@
-# Compass
-
-Install Compass and incidentally Sass via `apt-get`. The Ruby role is installed as dependency.
-
 # Gulp
 
+_Existing configuration files will not be overriden._
```

Comment:
Could you also mention which files will be overridden or not? For example what if my project already has a `Gulpfile.js` file?